Standards/scs-XXXX-vN-decision-record-template.md

```diff
@@ -0,0 +1,41 @@
+---
+title: _Descriptive title_
+type: Decision Record
+status: Draft
+track: Global  # | IaaS | Ops | KaaS | IAM
+author: Author <Mail or Github handle>
+date: DD-MM-YYYY
+---
+
+<!---
+This is a template striving to provide a starting point for
+creating a standard or decision record adhering to scs-0001.
+Replace at least all text which is _italic_.
```

Comment:
```suggestion
Replace at least all text in the sections not marked as OPTIONAL.
```

(there's no italic text in this template)

Reply:
https://github.com/SovereignCloudStack/standards/pull/441/commits/333f64e6cce05cceff6e49087cb595e310ab882f